source/standards/pull-requests.html.md.erb

```diff
@@ -1,166 +1,143 @@
-  otherwise not be involved (we can even 'mention' specific people with `@name`
-  syntax)
+- Enable [code review](/manuals/code-review-guidelines.html) from colleagues
+- Provide a lightweight change management process
+- Share responsibility for the code going live
+- Spread knowledge and make others aware of changes
+- Notify people outside the immediate group of reviewers
 
 ## Cautionary notes
 
-You should not:
+You should not use PRs for architectural review, as you should do this before you raise the pull request.
 
-- use PRs for architectural review - this should be done before you raise the pull
-  request
-- rely on someone else to spot and merge your PR - it's your job to find a
-  reviewer and get the code merged
+It is your responsibility to make sure your PRs are reviewed and merged. You should not rely on someone else noticing, reviewing, and merging your PR for you.
 
-A PR does not have to be all of the work - it's okay to have follow-up PRs for a
-  feature if a set of changes are complete enough to go live.
+You do not need to merge all changes for a piece of work in a single PR. You should break down the changes into several PRs to get feedback earlier.
 
 ## Guidance for each step
 
-If you're not sure how to do any of this, feel free to ask for help.
+Feel free to ask for help on Slack (for example, [#tech-community](https://gds.slack.com/archives/CACV4GHCL)) if you are unsure how to do any of these things.
 
 ### Opening a pull request
 
-Before opening the pull request, make sure:
-
-- you're up to date with `main` so that your
-  changes are easier to merge
-- the title and description help the reviewer
-- the title is succinct
-  and descriptive, with detail in the description
-- the description summarises your changes and includes useful links, for example, to
-  a Pivotal ticket, ZenDesk ticket or related PR
-- to describe any screenshots you use (for example, to show frontend changes) in the text, for accessibility reasons
-- the title and description are accurate and relevant, as GitHub emails these (but not any subsequent changes) to people following
-  the repo
-- you explain or highlight any potentially contentious changes, and
-  any testing that you have already done
-
-The canonical description of changes should always be in the individual
-commits - pull requests are an artefact of GitHub, and we would lose that data
-if we switched away. You can refer to the [commit message style
-guide](/standards/git.html).
+You should make sure that:
+
+- Your local repository has the most recent changes on the target branch
+- The title and description of the PR are clear and accurately represent your changes
+- The title and description reference the source of the change. This could be a ZenDesk ticket or a card in JIRA
+- The description contains links to any related PRs  
+- Any screenshots have text descriptions for accessibility
+- Any contentious changes or side effects of your change should be highlighted
```

Comment:
```suggestion
- Any contentious changes or side effects of your change are highlighted
```

Reply:
Ack'd, I changed this to remove the passive voice and to draw attention to the need for explanations of pros/cons.